scipy/stats/_continuous_distns.py

```diff
@@ -7361,6 +7657,9 @@ def _rvs(self, c, size=None, random_state=None):
     def _argcheck(self, c):
         return (c >= 0) & (c <= 1)
 
+    def _shape_info(self):
+        return [_ShapeInfo("c", False, (0, 1.0), (True, True))]
```

Comment:
Would be nice if docs mentioned bounds on `c` explicitly, but we can leave such things to a cleanup effort.